src/sass/_uswds-theme-custom-styles.scss

```diff
@@ -97,3 +97,158 @@ i.e.
     max-width: 50%;
   }
 }
+
+// Buttons
+.usa-button {
+  height:units(6);
+  padding: units(2) units(2.5);
+  &.button-icon {
+    padding-top:units(1.5);
+    padding-bottom:0.875rem;
+  }
+}
+
+.button-icon svg {
+    vertical-align: sub;
+    margin-bottom: -2px;
+  }
+
+.usa-button:focus,
+.usa-button.usa-focus {
+  outline-color: color($theme-color-primary-vivid);
+}
+
+//Primary Buttons (dark)
+.usa-button.primary-button-dark:not(:disabled) {
+  color: color($theme-text-color);
+  background-color: color($theme-color-base-lightest);
+}
+
+.usa-button.primary-button-dark.usa-button--hover:not(:disabled),
```

Comment:
Had to make some custom classes (e.g. primary-button-dark) since there was no corresponding dark mode solid button